app/lib/ability.rb

```diff
@@ -8,6 +8,8 @@ def initialize(user)
 
     if user.is_admin?
       can :manage, :all
+    elsif user.is_client_support?
```

Comment:
If the `Ability` class gets much longer, consider separating it out into different Ability classes per role. Since a user can only have one role, this might make it easier to unit-test each role's abilities (one unit test per ability file) and then express more specific permissions.

Not a blocker for this pull request.

Here's a long discussion explaining how to do that: https://github.com/CanCanCommunity/cancancan/issues/352

Here's some sample code copy-pasted from there to make the idea hopefully clearer. First, modify ApplicationController so the `current_ability` method returns a different class based on the user's role. Copy-pasting from that discussion:

```
class ApplicationController < ActionController::Base
   def current_company
         ...
   end 
 private
    def current_ability
      @current_ability ||=
        if current_user
          Ability.new(current_user, current_company)
        else current_super_admin
          SuperAdminAbility.new
        end
    end
end
```

Then we'd implement different classes -- not `Ability` and `SuperAdminAbility` like in the copy-pasted code, but maybe `AdminAbility` and `ClientSupportAbility`, since those are two roles we'd have.

Not a blocker for this PR, but something to consider.